### PR TITLE
Add support providers page and tweak some general aspects of the website

### PR DESCRIPTION
**Pull request by galexrt**

### CODE REVIEW

**Review by jbw976 — COMMENTED** [review]
@galexrt is there a preview site deployment for this change that we can browse and get a visual sense for this change? 👀

File: support.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                    <li>Hosted Rook Ceph clusters.</li>
+                    <li>Data Control Center application for insights into Rook Ceph clusters.</li>
+                </ul>
+            </p>
+        </section>
+
+        <section class="col-12_md-12">
+            <h2><a href="{{ redHatLink }}">Red Hat</a></h2>
+            <p>
+                Red Hat® OpenShift® Data Foundation—previously Red Hat OpenShift Container Storage—is software-defined storage for containers. Red Hat OpenShift Data Foundation helps teams develop and deploy applications quickly and efficiently across clouds.
```
Comment (travisn) [inline support.html:46]:
```suggestion
                Red Hat® OpenShift® Data Foundation is software-defined storage for containers. Red Hat OpenShift Data Foundation helps teams develop and deploy applications with storage quickly and efficiently across clouds.
```

File: support.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+                Red Hat OpenShift Data Foundation is included with Red Hat OpenShift Platform Plus, a complete set of powerful, optimized tools to secure, protect, and manage your apps.
+
+                <ul>
+                    <li>OpenShift Data foundation is based on Rook and other open source projects.</li>
```
Comment (travisn) [inline support.html:51]:
```suggestion
                    <li>Based on Rook, Ceph, and other open source projects</li>
                    <li>Fully supported Enterprise solution</li>
```

File: support.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                </ul>
+            </p>
+        </section>
+    </div>
+
+    <section class="grid-middle">
+        <div class="col-12_md-12">
+            <p>
+                If you want to add your company to the list of support providers, please open a pull request against the <a href="https://github.com/rook/rook.github.io">rook.github.io repository</a>.
```
Comment (travisn) [inline support.html:60]:
```suggestion
                To add your company to the list of support providers, please open a pull request against the <a href="https://github.com/rook/rook.github.io">rook.github.io repository</a>.
```

File: support.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+---
+
+{% include values.inc %}
+
+<div id="holder">
+    <section class="grid-middle hero">
+        <div class="col-5_md-12">
+            <h1>Support</h1>
+            <p>
+                You can ask questions in our community forums in <a href="{{ slackLink }}">Slack</a> or <a href="{{ slackLink }}">GitHub Discussions</a>, both of which have many knowledgeable users.
```
Comment (travisn) [inline support.html:12]: How about making this a bullet list? Something like this:
```
The Rook community is here to support you:
- Ask questions in our community forums in <a href="{{ slackLink }}">Slack</a> or <a href="{{ githubDiscussionsLink }}">GitHub Discussions</a>
- To request a feature or report a defect, <a href="{{ newIssueLink }}">open a GitHub Issue</a>
- To contribute to the project, see the <a href="{{ contributeLink }}">Contribution Guide</a>
- For commercial support check out the <a href="#support-companies">list of Rook-related support and services companies</a> below.
```

Note we need to define `githubDiscussionsLink`

File: support.html [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        <div class="col-12_md-12">
+            <p>
+                If you want to add your company to the list of support providers, please open a pull request against the <a href="https://github.com/rook/rook.github.io">rook.github.io repository</a>.
+            </p>
+        </div>
+    </section>
+
+    <section class="grid-middle text-center">
+        <div class="col-12_md-12">
+            <h1>Reporting Bugs</h1>
```
Comment (travisn) [inline support.html:67]: No need for this section if we go with my suggested bullet list above

**Comment by galexrt** [comment]
> @galexrt is there a preview site deployment for this change that we can browse and get a visual sense for this change? 👀

@jbw976 sadly not that I know of, but you can run `make run` locally on the PR branch.

**Comment by galexrt** [comment]
@travisn Here is a screenshot of the latest support page with the changes 
image

@jbw976 Here's a video with the changes highlighted, only thing outdated now is the "Support" page (see screenshot above).


https://github.com/rook/rook.github.io/assets/3718398/28bdac72-9db6-450d-95f0-d2cda4aa7baa

**Review by travisn — APPROVED** [review]
LGTM!

**Review by jbw976 — APPROVED** [review]
Awesome, thanks for taking this on @galexrt!! this looks reasonable to me ✅

**Pull request merged**